pkg/devfile/parser/parse.go

```diff
@@ -21,22 +47,28 @@ func Parse(path string) (d DevfileObj, err error) {
 		return d, err
 	}
 
-	// Validate devfile
-	err = d.Ctx.Validate()
+	d, err = ParseDevfile(d)
```

Comment:
Can be simplified to only `return ParseDevfile(d)`

Reply:
Done